livebook/parsers/scene.py

```diff
@@ -0,0 +1,21 @@
+class Scene:
+
+    def __init__(self, text, options):
+        self.text = text
+        self.options = options
```

Comment:
Do we insist that for each scene options should be distinct?

Reply:
No. No real reason to forbid non-distinct options.